docs/src/pages/css-in-js/basics/basics.md

```diff
@@ -52,6 +52,19 @@ install();
 ```
 
 It is **recommended** to place the above code in a separate file (e.g. `bootstrap.js`) and to import it in your application's entry point (e.g. `index.js`).
+
+This bootstrap file should be imported before any of your own components that use Material-UI components. 
+
+eg.
```

Comment:
'e.g.' 

Also, the preceding sentence is a bit awkward, and could have been reworded slightly before merge. At least this section is temporary...

Reply:
Yes, it will soon be gone :)